spec/smarter_csv/simple_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
```

Comment:
put this into `.rspec` [instead](https://github.com/rspec/rspec-core/blob/main/README.md#store-command-line-options-rspec)

```
> cat .rspec 
--require spec_helper
```